factory/processing/processComponents.go

```diff
@@ -1515,16 +1515,28 @@ func (pcf *processComponentsFactory) newStorageRequesters() (dataRetriever.Reque
 		}
 	}()
 
+	clonedConfig := pcf.config
+	clonedConfig.AccountsTrieStorage = config.StorageConfig{
+		Cache: config.CacheConfig{
+			Type:     string(storageunit.LRUCache),
+			Capacity: 10,
```

Comment:
why do we hardcode the importdb trie storage configs?
since we have now hardcoded the db as memorydb, the importdb will fail if we have large tries and running the import on machines without sufficient ram.

I would add a config for this so that it is possible to run it on other machines as well